lang/src/main/kotlin/org/partiql/lang/eval/physical/EvaluatorState.kt

```diff
@@ -31,6 +31,7 @@ class EvaluatorState(
     /** The current [EvaluationSession]. */
     val session: EvaluationSession,
 
+    @Deprecated("[ExprValueFactory] is deprecated")
```

Comment:
Could you make a new primary constructor without the valueFactory parameter? Then update the PhysicalPlanCompilerImpl to use the new?

Reply:
Will apply